install/kubernetes/cilium/charts/hubble-relay/templates/deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
                   values:
                     - cilium
             topologyKey: "kubernetes.io/hostname"
+      serviceAccountName: hubble-relay
```

Comment:
One more change for old kube version. Can you help to add `serviceAccount: hubble-relay` as well ?

Reply:
Similar to https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/charts/agent/templates/daemonset.yaml#L353